pkg/intermediate/aggregate.go

```diff
@@ -366,6 +366,16 @@ func (a *AggregationProcess) addOrUpdateRecordInMap(flowKey *FlowKey, record ent
 				}
 			}
 		} else {
+			// For inter-node traffic with deny np or drop anp, we may receive the record from the conntrack table first.
```

Comment:
this should be in a different PR, it's unrelated to the collector changes and seems like a bug fix to me